docs/helm/examples/with-ms-teams.md

```diff
@@ -41,7 +41,8 @@ helm install acs alfresco/alfresco-content-services \
 --set messageBroker.user="alfresco" \
 --set messageBroker.password="YOUR-MQ-PASSWORD" \
 --set msTeams.enabled=true \
---set msTeamsService.alfresco.baseUrl="https://acs.YOUR-DOMAIN-NAME:443"
+--set msTeamsService.alfresco.baseUrl="https://acs.YOUR-DOMAIN-NAME:443" \
```

Comment:
Didn't we want to put `http://localhost:8080` in `alfresco.baseUrl` when thought about introducing `alfresco.digitalWorkspace.baseUrl`?

Reply:
Yes. I think here, it could be the default Kubernetes internal url, eg.

https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/values.yaml#L119

`http://acs-alfresco-cs-repository:80`

However, as per [team review](https://github.com/Alfresco/ms-teams-integration/pull/43) (private repo) we'd need both internal and external urls for different usages.